### DISCUSSION TITLE
Social Cards - background transparent+image not working

### DISCUSSION THREAD

**Discussion by Siroccomeister** [squidfunk/mkdocs-material#7391] [Q&A] [answered]
Hi - I have tried many options but for some reason it still is not working. I followed the documentation for social cards as I am about to ride across the USA (Canada >> Mexico - GDMBR route) and wanted to blog with Material MKDOCS.
- social plugin is working
- background colour option is picked up in hex code (e.g. pink)
- png gets built with GitHub CI
BUT when I use "transparent" : CI build triggers an error of unknown color
IF I try to use a hex code "00FFFFF" or "FFFFF00" and add an image in layouts at project root level (same as mkdocs.yml), the image never gets visible.

I thought these features were available outside of insider's edition but I am not too sure now. Anyone could help/advise? I am running out of time to get all the details sorted before starting my adventure in 2 days...

Many thanks !

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. The [`background_image` option](https://squidfunk.github.io/mkdocs-material/plugins/social/#option.background_image) is currently only available in Insiders.

**Reply by Siroccomeister** [reply]
Ok many thanks. This explains it. I got confused as the image option is described in the basic social cards before the insider's customisation. So I tested all possible options to make it work... fully understand it is challenging to handle all these dependencies. Thanks With best regards!

https://squidfunk.github.io/mkdocs-material/tutorials/social/basic/?h=basic
